src/Umbraco.Core/IO/IOHelper.cs

```diff
@@ -13,7 +13,9 @@
 namespace Umbraco.Core.IO
 {
 	public static class IOHelper
-    {
+	{
+	    public static bool IAmUnitTestingSoNeverUseHttpContextEver = false;

```

Comment:
See https://github.com/lars-erik/specflow-razorgenerator-umbraco-samples for context.